### PR TITLE
Update to Vite 5

### PR DESCRIPTION
**Pull request by giacomorebonato**
All tests passing

Screenshot 2023-11-27 at 13 31 07

### CODE REVIEW

**Comment by giacomorebonato** [comment]
@cyco130 do you think that I could ask for a review of this?

Thanks for this awesome tool

File: packages/vavite/tsup.config.bundled_v10kr34fv7.mjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// tsup.config.ts
```
Comment (TylorS) [inline packages/vavite/tsup.config.bundled_v10kr34fv7.mjs:1]: I"m not a maintainer/contributor (sorry), but I'd suspect this file should be deleted from the diff

**Comment by giacomorebonato** [comment]
I created a fork, if you want to try it out and contribute @TylorS 

* https://www.npmjs.com/package/@giacomorebonato/vavite

I don't mean to be unpolite to this great project in any way and it's the first time that I fork something published on npm:

* I am happy to deprecate the fork and contribute here if it's welcome
* if I didn't give the right credit to @cyco130 , please reach out and I'll address that. That's not intentional.

**Comment by TylorS** [comment]
It took me a little while to get back to working on abstractions related to vavite, and I tested out your fork @giacomorebonato, and so far after about an hour of using it I haven't run into any issues and everything seems to be working great

**Comment by cyco130** [comment]
Sorry for the late response and thank you for your contribution. I will release soon!

**Pull request merged**